src/components/CMultiOptionButton.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-menu :target="buttonGroup">
+    <template #activator="{ props: activatorProps }">
+      <v-btn-group
+        ref="buttonGroup"
+        block
+        :color="color"
+        :variant="variant"
+        :density="density"
+        rounded="xl"
+      >
+        <v-btn
+          :to="primaryButton.to"
+          :disabled="disabled"
+          :loading="loading"
+          @click="primaryButton.onClick"
+        >
+          {{ primaryButton.label }}
+        </v-btn>
+
+        <v-btn
+          v-bind="activatorProps"
+          :icon="mdiChevronDown"
+          :disabled="disabled"
```

Comment:
```suggestion
          :disabled="disabled || loading"
```
?

Reply:
i didn't want to change the behaviour of `loading` or `disabled` from vuetify at this point. Do you think we should?

Reply:
Well, the left portion of the button automatically gets disabled by the `loading` prop. I think it is kind of odd to have the dropdown activated. 

But I do get your point, we can also decide whereever this button is implemented if we want to disable it while loading, can we?

Reply:
Ah I see, what you mean. Vuetify does "disable" the button when loading, but not visually. I think it would be stranger if we then only visually disabled the right button, so we should disable both when loading and change the behaviour in general?

Reply:
Looks good